### ISSUE TITLE
[Suggestion]: FTBTeams / FTBQuests - Disable Team Quest Progression

### ISSUE THREAD

**Issue by 1Diamond1** [FTBTeam/FTB-Modpack-Issues#1678]
### Modpack

StoneBlock 3

### Is your suggestion related to a problem? Please describe.

My friend doesn't want to share quest rewards/progression but doesn't want to separate between worlds. I believe this was implemented in the previous StoneBlock's FTBTeams though. Any feedback would be appreciated on how I can work around the team system and not be in the same team while in the same world. This **_is_** on a server, not a LAN or anything like that.
Any suggestion is, again, appreciated.

**Comment by Raidobw2** [comment]
I believe one could join their own team but teleport to you with `/tpa YourMinecraftName`. Then they could `/sethome` to your place and play with you (use `/home` to go back to your home). You might have to change your team settings to allow them to place blocks and interact with blocks.

I don't know a good solution to playing as a team without sharing progress, so the method I described above for example doesn't allow the 2 people in the same team to claim chunks or chunkload as if they had one team, because there are two.

I've labeled this as Documentation as maybe this could be in the Quest book to guide new players that want the same experience with their friends as well
